src/plugins/AutoRefresh.ts

```diff
@@ -47,7 +49,7 @@ class AutoRefresh extends GenericPlugin {
       return;
     }
 
-    const { refreshRateInSeconds = 30 } = this.ad.configuration;
+    const { refreshRateInSeconds = 30, refreshLimit = null } = this.ad.configuration;
```

Comment:
This needs to be added to the IAdConfiguration type in types file as well please